Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml

```diff
@@ -0,0 +1,160 @@
+          | project-rename LogMessage =  SyslogMessage
+          )
+      | where not(disabled)
+          and LogMessage has "events"
+          and (LogMessage has_any (LogSubTypeList) or LogMessage has_all("disassociation","auth_neg_failed"))
+      | extend Parser = extract_all(@"(\d+.\d+)\s([\w\-\_]+)\s([\w\-\_]+)\s([\S\s]+)$", dynamic([1, 2, 3, 4]), LogMessage)[0]
+      | extend
+          Epoch = tostring(Parser[0]),
+          Device = tostring(Parser[1]),
+          LogType = tostring(Parser[2]),
+          Substring = tostring(Parser[3])
+      | where LogType == "events"
+      | parse Substring with * "type=" LogSubType:string " " restOfMessage:string
+      | where LogSubType in (LogSubTypeList) or (LogSubType == "disassociation" and Substring has "auth_neg_failed")
+      | extend EpochTimestamp = split(Epoch, ".")
+      | extend EventStartTime = unixtime_seconds_todatetime(tolong(EpochTimestamp[0]))
+      | extend EventEndTime = EventStartTime
+      | invoke _ASIM_ResolveDvcFQDN('Device')
+      | parse-kv Substring as(last_known_client_ip: string, ip: string, client_ip: string, client_mac: string, identity: string, reason: string, aid: string) with (pair_delimiter=" ", kv_delimiter="=", quote="'")
+      | extend Dvc = DvcHostname, 
+            aid = trim('"', aid)
+      | extend
+          SrcIpAddr = tostring(split(coalesce(last_known_client_ip, ip, client_ip), " ")[0]),
+          DvcMacAddr = client_mac,
+          User = identity,
+          AdditionalFields = bag_pack("aid", aid),
+          EventOriginalType = LogType,
+          EventOriginalSubType = LogSubType
+      | extend
+          SrcIpAddr = trim('"', SrcIpAddr),
+          DvcMacAddr = trim('"', DvcMacAddr),
+          User = trim('"', User),
+          reason = trim('"', reason)
+      | extend
+          DvcIpAddr = SrcIpAddr,
+          IpAddr = SrcIpAddr
+      | lookup EventFieldsLookup on LogSubType
+      | lookup EventResultDetailsLookup on reason
+      | extend EventResultDetails = iff(tolong(reason) between (25 .. 65535), "Other", EventResultDetails)
+      | extend
+          EventCount=int(1),
+          EventProduct="Meraki",
+          EventVendor="Cisco",
+          EventSchema="Authentication",
+          EventSchemaVersion="0.1.3"
+      | project-away
+          LogMessage,
+          Parser,
+          Epoch,
+          EpochTimestamp,
+          Device,
+          Substring,
+          LogType,
+          LogSubType,
+          restOfMessage,
+          reason,
+          last_known_client_ip,
+          client_ip,
+          ip,
+          client_mac,
+          identity,
+          aid,
+          TenantId,
+          SourceSystem,
+          Computer,
+          _ResourceId,
+          MG,
+          ManagementGroupName,
+          RawData,
+          EventTime,
+          Facility,
+          HostName,
+          SeverityLevel,
+          ProcessID,
+          HostIP,
+          ProcessName,
+          CollectorHostName
```

Comment:
@jayeshprajapaticrest  try adding this to the meraki custom table. This column is showing in the validation error

Reply:
@vakohl CollectorHostName column is a part of Syslog table which is added recently in [Syslog table schema.](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/syslog) So we have added that in project-away.
![image](https://github.com/Azure/Azure-Sentinel/assets/98145046/b695fd79-7343-410d-b99b-9f6aa44f3933)